fedora_elections/templates/index.html

```diff
@@ -32,6 +32,10 @@
               <a href="{{ url_for('vote', election_alias=election.alias) }}">
               Vote now!
               </a>
+            {% else %}
```

Comment:
Should we check the election type here for now? (to avoid offering to revote on a type that doesn't support revote yet)